doc/setup/deploy.rst

```diff
@@ -544,8 +547,35 @@ Here is what it could look like for blocking the IP for 15 minutes when
     findtime = 60  ; within 1 min  /!\ Should be adjusted with the TZ offset
     logpath = /var/log/odoo.log  ;  set the actual odoo log path here
 
+Scenario 2) Odoo works behind a proxy, like Apache.
+In this case we cannot use the Odoo logs, as no valid IP will be logged. But the proxy server (e.g. Apache) knows the IPs where the request originated from. A failed login attempt will cause Odoo to POST the page /web/login to the user again (and again ...). And this is logged by the proxy server! A failed login on Apache looks something like:
 
+    7.120.35.25 - - [13/Jan/2019:22:52:25] "POST /web/login HTTP/1.1" 200 2514 "https://YOUR-DOMAIN.com/" "Mozilla/5.0 (X11; Ubuntu; ....
+    
+This log entry can be analyzed and counted by `fail2ban`. The following fail2ban filter definition should match a failed login on Apache:
 
+    failregex = <HOST> .* /web/login .*
```

Comment:
wouldn't it be better to have `<HOST> .*POST /web/login .*` here? (to only match requests that sent a username and password and not those that just displayed the login form without trying to login)

Reply:
Yes I think this would be even better!

Reply:
I implemented your comment to the PR.